Library/Homebrew/cmd/shellenv.sh

```diff
@@ -57,7 +57,7 @@ homebrew-shellenv() {
       echo "export HOMEBREW_CELLAR=\"${HOMEBREW_CELLAR}\";"
       echo "export HOMEBREW_REPOSITORY=\"${HOMEBREW_REPOSITORY}\";"
       echo "export PATH=\"${HOMEBREW_PREFIX}/bin:${HOMEBREW_PREFIX}/sbin\${PATH+:\$PATH}\";"
-      echo "export MANPATH=\"${HOMEBREW_PREFIX}/share/man\${MANPATH+:\$MANPATH}:\";"
+      echo "[ -n \"\${MANPATH}\" ] && export MANPATH=\":\${MANPATH#:}\";"
```

Comment:
```suggestion
      echo "[ -z \"\${MANPATH}\" ] || export MANPATH=\":\${MANPATH#:}\";"
```

might be better because it'll handle `set -e` better

Reply:
Good catch, thanks @MikeMcQuaid!